xarray/tutorial.py

```diff
@@ -85,15 +80,9 @@ def open_dataset(
     except ImportError:
         raise ImportError("using the tutorial data requires pooch")
```

Comment:
is `ImportError` the right exception to raise here? Also, would it be better to silence the original exception?